net/onioncat/Portfile

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                onioncat
+version             0.2.2.r569
+categories          net
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3
+
+description         VPN-adapter that provides VPN-tunnels using Tor or I2P
+
+long_description    OnionCat is a VPN-adapter which allows to connect two or \
+                    more computers or networks through VPN-tunnels. It is \
+                    designed to use the anonymization networks Tor or I2P \
+                    as its transport, hence, it provides location-based \
+                    anonymity while still creating tunnel end points \
+                    with private unique IP addresses.
+
+homepage            https://www.onioncat.org/
+master_sites        https://www.cypherpunk.at/ocat/download/Source/current/
```

Comment:
This is OK, but just a minor thing. It would probably be slightly more "future-proof" to use the actual version here. When the software switches to 0.2.3, this URL will most likely no longer serve the sources for 0.2.2. Of course you should upgrade the software soon after that.

I didn't test the exact syntax, but you could use something in the following spirit:
```tcl
set branch          [join [lrange [split ${version} .] 0 2] .]
master_sites        https://www.cypherpunk.at/ocat/download/Source/${branch}/
```